components/org.wso2.carbon.appmgt.api/src/main/java/org/wso2/carbon/appmgt/api/model/WebApp.java

```diff
@@ -115,6 +115,13 @@
     //TODO: missing - total user count, up time statistics,tier
 
 
+    public WebApp() {
+    }
+
```

Comment:
Since we are having overload constructor in a class it is must to have default constructor as well.